src/s2japp/ApplicationSettings.cc

```diff
@@ -23,7 +23,7 @@ namespace {
  */
 class StringOutputGenerator : public TCLAP::StdOutput {
 public:
-    StringOutputGenerator(s2j::app::ApplicationSettings& settings)
+    explicit StringOutputGenerator(s2j::app::ApplicationSettings& settings)
```

Comment:
to robi, żeby to nie była implicitna konwersja, tak?
Na pewno nie trzeba tego też gdzieś indziej?

Reply:
Może trzeba, sam clang-tidy tego już nie poprawił - ja teraz nie szukałem.

Reply:
Ok, ale czy jest jakiś powód dla którego tutaj to jest potrzebne a gdzie indziej nie?
Jeśli nie, to czemu nie robimy wszystkich konstruktorów wszystkich klas `excplicit`?